src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
```

Comment:
let's try to not use Data annotaion for entities. Better to use Getter Setter EqualsAndHashCode etc.
please check other entities.